go-controller/pkg/node/port_claim.go

```diff
@@ -73,38 +80,61 @@ func initPortClaimWatcher(recorder record.EventRecorder, wf *factory.WatchFactor
 }
 
 func addServicePortClaim(svc *kapi.Service) []error {
-	return handleService(svc, port.open)
+	return handleService(svc, portHandler.open)
 }
 
 func deleteServicePortClaim(svc *kapi.Service) []error {
-	return handleService(svc, port.close)
+	return handleService(svc, portHandler.close)
 }
 
 func handleService(svc *kapi.Service, handler handler) []error {
 	errors := []error{}
 	if !util.ServiceTypeHasNodePort(svc) && len(svc.Spec.ExternalIPs) == 0 {
 		return errors
 	}
+
 	for _, svcPort := range svc.Spec.Ports {
 		if util.ServiceTypeHasNodePort(svc) {
-			if err := handlePort(svc, svcPort.NodePort, svcPort.Protocol, handler); err != nil {
+			klog.Warningf("Handle NodePort service %s port %d", svc.Name, svcPort.NodePort)
```

Comment:
It doesn't seem to me like this should be logged as a warning?

Reply:
changed to klog.V(5).Infof(...)